libcloud/compute/drivers/runabove.py

```diff
@@ -104,6 +126,15 @@ def create_node(self, **kwargs):
         return self._to_node(response.object)
 
     def destroy_node(self, node):
+        """
```

Comment:
Docstrings for methods which are part of the standard API are not required (unless you changed the method signature) since they are inherited from the parent class.